frontend/src/components/mine/MineDashboard.js

```diff
@@ -123,7 +125,8 @@ export class MineDashboard extends Component {
 
   loadMineData(id) {
     this.props.fetchMineRecordById(id).then(() => {
-      this.props.fetchApplications({ mine_guid: this.props.mines[id].guid });
+      this.props.fetchApplications({ mine_guid: this.props.mines[id].mine_guid });
+      this.props.fetchPermits({ mine_guid: this.props.mines[id].mine_guid });
```

Comment:
We were fetching the permits on there own for the permits tab so I brought it out to the dashboard so that the permits could be fetched for the summary tab and remove most of theyre information from the mine object.

Reply:
Cool. Marking as resolved -- we can carefully confirm in manual testing that this doesn't prevent permits from being loaded on any views